### PR TITLE
swastika-hometown: Stubbing out a file for Mountain View

### LINKED ISSUE #36: Create Swastika hometown file
## Mountain View, CA
- [x] Create a branch
- [x] Create a file
- [x] Add a commit
- [x] Create a Pull Request
- [ ] Have a conversation
- [ ] Get a 👍 
- [ ] 🚢 it (a.k.a. merge)

### PR DESCRIPTION
**Pull request by swastika**
This PR resolves #36

### CODE REVIEW

File: swastika-mtv.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+## Mountain View, CA
+
+### Places to eat
```
Comment (peter-sandor-1992) [inline swastika-mtv.md:3]: what's the city's population? :smiley:

Comment (githubteacher) [inline swastika-mtv.md:3]: I will be in Mountain View in a few weeks! Any recommendations for great Italian @swastika ?

Comment (swastika) [inline swastika-mtv.md:3]: @githubteacher Maggiano's Little Italy in San Jose is great! Don't know about a great Italian restaurants in Mountain View :(

**Comment by swastika** [comment]
This looks good. 👍

**Pull request merged**